features/step_definitions/cucumber_rails_steps.rb

```diff
@@ -58,19 +56,26 @@ def gem(name, *args)
   end
 
   def prepare_aruba_report
-    if ENV['ARUBA_REPORT_DIR']
-      @aruba_report_start = Time.new
-      sleep(1)
-    end
+    return unless ENV['ARUBA_REPORT_DIR']
+
+    @aruba_report_start = Time.new
+    sleep(1)
   end
 
   def fixture(path)
     File.expand_path(File.dirname(__FILE__) + "./../support/fixtures/#{path}")
   end
+
+  private
+
+  def rails6?
+    `bundle exec rails -v`.split("\n").last.strip.start_with?('Rails 6')
```

Comment:
Can one use  `.lines.last` here?

```suggestion
    `bundle exec rails -v`.lines.last.strip.start_with?('Rails 6')
```

Reply:
Probably, I'll make the change and then go with it. For this manual one I just copied and pasted what was there beforehand.